.gitignore

```diff
@@ -1,4 +1,7 @@
 .DS_STORE
 node_modules
 bower_components
-build
\ No newline at end of file
+build
+public/scripts/main/misc/filters.js
```

Comment:
Why did you add to remove 2/3 of the core critical components of the Angular app? Please make sure to do a full review of what you push before you push it - these sort of pushes are dangerous. My assumption is that you are just doing `git add *` or `.` or something that just adds everything in a wanton fashion or that you are using the GUI. Whatever is happening - it is creating a number of serious externalities which we want to avoid. Please read https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging